### ISSUE TITLE
Where is stylesheet defined

### ISSUE THREAD

**Issue by annevk** [w3c/csswg-drafts#6101]
CSS Syntax says it creates a stylesheet, but it doesn't reference this concept or mention all the associated concepts it has.

One way this comes up is quirks mode (see #6100), but there's also be the base URL and various other things. HTML and CSSOM go into this to some extent as well, but having this more centralized or at the very least linked in a way that makes sense would help tremendously.

**Comment by tabatkins** [comment]
Yeah, I don't link to the concept from Syntax because there wasn't a good anchor to link to. :/
